cmd/internal/kube/inject.go

```diff
@@ -28,13 +29,29 @@ var (
 	// When set to "true", injectCmd will prepend a secret to each injectable namespace found in the file to inject (injectFileNameFlag)
 	// Otherwise, injectCmd will treat secretInjectFlag as the file path all secrets should be generated to
 	secretInjectFlag string
+
+	// Postman related flags
+	postmanCollectionID string
+	postmanAPIKey       string
+	postmanEnvironment  string
 )
 
 var injectCmd = &cobra.Command{
 	Use:   "inject",
 	Short: "Inject Akita into a Kubernetes deployment",
 	Long:  "Inject Akita into a Kubernetes deployment or set of deployments, and output the result to stdout or a file",
 	RunE: func(_ *cobra.Command, args []string) error {
+		if projectNameFlag == "" && postmanCollectionID == "" {
+			return cmderr.AkitaErr{
+				Err: errors.New("exactly one of --project or --collection must be specified"),
+			}
+		}
+
+		// Adding postman API key and Environment in config for further uses
+		if postmanCollectionID != "" {
+			cfg.WritePostmanAPIKeyAndEnvironment("default", postmanAPIKey, postmanEnvironment)
```

Comment:
I think we need to avoid writing the config file here too.

Reply:
Currently, I am trying to read both Postman API Key and Environment from the config file, is it okay? Or, should we keep only the API key in the config file?
Also I am thinking to only write postman environment to config file when it is sent by user, cause anyhow in production it will never be used.